### ISSUE TITLE
MetadataJson Delayed Start

### ISSUE THREAD

**Issue by BlakeItOut** [aws-samples/amazon-transcribe-live-call-analytics#181]
The delayed start Eventbridge event should contain an optional field for updating the MetadataJson. The lambda that handles this event would need to be updated to dynamically add the MetadataJson.

**Comment by rstrahan** [comment]
Good idea @BlakeItOut. Can you elaborate on the use cases that this feature would facilitate?
Are you in a position to make the code changes in a fork, and send a PR? (that would expedite things)
Cheers

**Comment by BlakeItOut** [comment]
Sure, we have a parallel system that gets metadata when chime starts a call so it is too slow for the filter lambda to get that metadata and the data does not exist on the original siprec data. This would allow us to add it as soon as we have it and then have it available for downstream systems that actually feed off of DynamoDB streams. We can make the PR.

**Comment by rstrahan** [comment]
Just checking in @BlakeItOut - is a PR for this still in the works?

**Comment by BlakeItOut** [comment]
Have not gotten a chance to do this yet. If this is a pressing need for anyone else or anyone wants to run ahead with this PR, happy to have someone else take the lead. Otherwise, it's still on my todo list.
